microsoft/PowerToys
# [Shortcut Guide] Telemetry

## Summary of the Pull Request

**What is this about:**

**What is include in the PR:** 
- Replace `ShortcutGuide_SettingsChanged` with `ShortcutGuide_Settings` event, follow general approach to send settings telemetry
- Remove `ShortcutGuide_EnableGuide` event. It should be sent with general settings or with the module settings
- Remove `ShortcutGuide_Error` event as it is not sent anymore
- Signal event to terminate SG process. It is needed for `ShortcutGuide_HideGuide` event. Probably, we should use another event name
- Drop remaining of  shortcut support for SG window

**How does someone test / validate:** 
- Press `Shift+Win`
- Start quickly typing `/`
- Verify log file for errors. It is possible that sometimes the SG window is not shown as the previous one hasn't completely closed yet.

## Quality Checklist


## Contributor License Agreement (CLA)
A CLA must be signed. If not, go over [here](https://cla.opensource.microsoft.com/microsoft/PowerToys) and sign the CLA.
